parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -219,7 +219,7 @@ downloadAzureCNI() {
 
 ensureAPMZ() {
```

Comment:
@xuto2 would it make sense to also exclude AKS from this install since it isn't using CSE telemetry?